core/storage/exploration/gae_models.py

```diff
@@ -32,6 +34,9 @@
 if MYPY: # pragma: no cover
     from mypy_imports import datastore_services  # pylint: disable=unused-import
 
+# Constants used for generating ids.
```

Comment:
Try to add a more descriptive adjective before "ID" perhaps, to distinguish from other types of IDs.

Also perhaps this should be scoped to the TransientCheckpointUrlModel class instead, if that's the only place it's used?

Reply:
Fixed.